content/browser/renderer_host/media/audio_sync_reader.cc

```diff
@@ -187,11 +190,15 @@ void AudioSyncReader::RequestMoreData(base::TimeDelta delay,
   }
 
 #if defined(CASTANETS)
-  size_t sent_bytes = HANDLE_EINTR(send(accept_handle_.get().handle, &control_signal, sizeof(control_signal), MSG_NOSIGNAL));
+  size_t sent_bytes = 0;
+#if !defined(OS_WIN)
+  HANDLE_EINTR(send(accept_handle_.get().handle, &control_signal,
```

Comment:
I think it is better to put the return value of send() in 'sent_bytes'.
sent_bytes = HANDLE_EINTR(send(accept_handle_.get().handle, ...

Reply:
Yes, its a mistake. Thanks for pointing.